source/scripting_v3/GTA/Camera/CinematicCameraDirector.cs

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright (C) 2015 crosire & kagikn & contributors
+// License: https://github.com/scripthookvdotnet/scripthookvdotnet#license
+//
+
+using GTA.Native;
+
+namespace GTA
+{
+	/// <summary>
+	/// Represents the cinematic camera director.
+	/// The cinematic director is responsible for idle, vehicle cinematic, vehicle first person, and nose cameras.
+	/// </summary>
+	public static class CinematicCameraDirector
+	{
+		/// <summary>
+		/// Invalidates the cinematic idle camera, restarting the associated idle counter.
+		/// </summary>
+		public static void InvalidateIdleCam() => Function.Call(Hash.INVALIDATE_IDLE_CAM);
+
+		/// <summary>
+		/// Invalidates the vehicle cinematic idle camera, restarting the associated idle counter.
+		/// </summary>
+		public static void InvalidateVehicleIdleCam() => Function.Call(Hash.INVALIDATE_CINEMATIC_VEHICLE_IDLE_MODE);
+
+		/// <summary>
+		/// Gets a value indicating whether a first person vehicle camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="CinematicCameraDirector"/> is a first person vehicle camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsFirstPersonVehicleCamRendering => Function.Call<bool>(Hash.IS_CINEMATIC_FIRST_PERSON_VEHICLE_INTERIOR_CAM_RENDERING);
```

Comment:
The word "Interior" is mandatory so script devs won't get confused IMO. I don't want to leave the ambiguity if there's "First Person Vehicle Hood" option. So, should be `IsFirstPersonVehicleCamRendering` rather than `IsFirstPersonVehicleCamRendering`. If you haven't be convinced, google like "gta first person hood camera" and you would realize there's people who thinks the hood cam is one of the first person cams.